parlai/mturk/core/dev/agents.py

```diff
@@ -90,76 +80,13 @@ def is_final(self):
             self.status == self.STATUS_DISCONNECT
             or self.status == self.STATUS_DONE
             or self.status == self.STATUS_PARTNER_DISCONNECT
-            or self.status == self.STATUS_PARTNER_DISCONNECT_EARLY
+            or
+            # TODO REMOVE THIS TYPE
```

Comment:
`PARTNER_DISCONNECT_EARLY` will be phased out in a followup, as it is against MTurk TOS. Instead we will put people back into the queue on failure.